__docs__/wonder-blocks-tooltip/tooltip.stories.tsx

```diff
@@ -306,6 +295,30 @@ Controlled.parameters = {
     },
 };
 
+export const WithStyle: StoryComponentType = () => {
+    return (
+        <View style={[styles.centered, styles.row]}>
+            <Tooltip
+                contentStyle={{
+                    color: Color.white,
+                    padding: Spacing.xLarge_32,
+                }}
+                content={`This is a styled tooltip.`}
+                backgroundColor="darkBlue"
```

Comment:
Optional suggestion: I think it might be helpful to have this tooltip always be open so that it's easier for devs to inspect.